.github/workflows/build.yml

```diff
@@ -30,9 +30,6 @@ jobs:
           # Only Elixir 1.14 supports OTP 25
           - otp: "25.3"
             elixir: "1.14.4"
```

Comment:
BTW In a separate PR it would be good to bum this to 1.15 and OTP 26.

Reply:
Good idea! I've created #72.